docs/guidance/pod-command.md

```diff
@@ -0,0 +1,148 @@
+# Specify container commands for Ray head/worker Pods
+You can execute commands on the head/worker pods at two timings:
```

Comment:
can you add some real scenarios for users to better understand why they need to run commands along with the cluster setup?

Reply:
I think Microsoft did similar way in the past. they like to run some code after cluster is running, and it has conflict with `--block` at that time. I think eventually, we determine to have the flag for different users.

Reply:
> can you add some real scenarios for users to better understand why they need to run commands along with the cluster setup?

https://ray-distributed.slack.com/archives/C02GFQ82JPM/p1675378764037199
https://ray-distributed.slack.com/archives/C02GFQ82JPM/p1669647595429959

The document has already provided some real-world scenarios based on the two slack threads above.

> I think Microsoft did similar way in the past. they like to run some code after cluster is running, and it has conflict with --block at that time. I think eventually, we determine to have the flag for different users.

https://github.com/ray-project/kuberay/pull/912#issuecomment-1436393712
Yes, `--block` has already been a part of Ray-Start-Parameters.